src/wwwroot/js/genpage/params.js

```diff
@@ -317,6 +332,18 @@ function genInputs(delay_final = false) {
                 }
                 resTrick();
             });
+            swapAspectRatioButton.addEventListener('click', (event) => {
+                event.preventDefault();
+                if (inputWidth.value && inputHeight.value) {
```

Comment:
... noticed late, but, what is this even checking? 0.o

Reply:
the if? you can leave the values of the inputs empty so I did that to check that both have something to swap

Reply:
you can what

Reply:
![image](https://github.com/user-attachments/assets/2d459c59-c795-4610-88fc-a993bdd0d8f0)

Reply:
that is wacky and dumb but also it still just works without the check even if you do that

Reply:
Oh I see